temba/flows/models.py

```diff
@@ -1539,16 +1539,13 @@ def start_ussd_flow(self, all_contact_ids, start_msg=None, extra=None, flow_star
 
                 step = self.add_step(run, entry_rule, is_start=True, arrived_on=timezone.now())
                 if entry_rule.is_ussd():
-                    # create an empty placeholder message
-                    msg = Msg(org=self.org, contact_id=contact_id, text='', id=0)
```

Comment:
Honestly I don't think we really want to support push anyways, so I'm fine removing support for push.